vcd/config_test.go

```diff
@@ -184,6 +184,7 @@ type TestConfig struct {
 		Certificate2Path           string `json:"certificate2Path,omitempty"`           // absolute path to pem file
 		Certificate2PrivateKeyPath string `json:"certificate2PrivateKeyPath,omitempty"` // absolute path to private key pem file
 		Certificate2Pass           string `json:"certificate2Pass,omitempty"`           // absolute path to pem file
+		RootCertificatePath        string `json:"rootCertificatePath,omitempty"`        // absolute path to pem file
```

Comment:
We should also update `sample_vcd_test_config.json`

Reply:
Added